interfaces/vrep/robots/YouBotVrepRobot.m

```diff
@@ -8,7 +8,7 @@
 %           >> vi.connect('127.0.0.1',19997);
 %           >> vrep_robot = YouBotVrepRobot("youBot", vi);
 %           >> vi.start_simulation();
-%           >> robot.get_q_from_vrep();
+%           >> robot.get_configuration_space_positions();
```

Comment:
Same as in the other files.

Reply:
Updated in [05873d5](https://github.com/dqrobotics/matlab/pull/108/commits/05873d58d643581beeb2140712c1e05b56cdc928).